### ISSUE TITLE
How do we upgrade?

### ISSUE THREAD

**Issue by SteveALee** [microsoft/playwright#12179]
The npm installation is more complex than usual and I want to upgrade to 1.19 for 1.18

Can you add instructions to the [installation](https://playwright.dev/docs/intro#installation) page?

Thanks.

**Comment by dgozman** [comment]
@SteveALee Did you try upgrading already? What did you do? Did you run any commands? Did you get any errors? I'd like to understand where the issue comes from to address it properly.

**Comment by SteveALee** [comment]
@dgozman fair point. I was perhaps overthinking and being over cautions. I've been down so many unwanted installation rabbit holes.

I ran the manual  `npm i -D @playwright/test` which seem to work with 1.18 -> 1.19 as expected given my limited understanding of semver and package-lock interactions (^ range specified).

But do I also need 'npx playwright install' which it seems may only update browsers?

And then that raised the question should I regularly 'npx playwright install' in order to keep the browses current. Or are they somehow evergreen?

Thanks

**Comment by dgozman** [comment]
@SteveALee You did the right thing for updating from 1.18 to 1.19.

`npx playwright install` is needed every time you update Playwright to update the browsers. Don't worry though, if you forget this, Playwright will complain next time you use it and will suggest to run `npx playwright install` in the error message. You can also run `npx playwright install --help` to get more information.

Browsers are directly tied to the Playwright version, so you need exactly one `npx playwright install` after upgrading Playwright, no need to run it regularly.
